spec/features/create_work_admin_spec.rb

```diff
@@ -28,17 +28,19 @@
     end
 
     it "allows default admin set to be the first item in the select menu" do
-      visit '/dashboard'
-      click_link 'Works'
-      click_link "Add new work"
-      choose "payload_concern", option: "GenericWork"
-      click_button 'Create work'
-      click_link "Relationship" # switch tab
-      expect(page).to have_content('Administrative Set')
-      expect(page).to have_content('Second Admin Set')
-      expect(page).to have_content('Default Admin Set')
-      expect(page).to have_selector('select#generic_work_admin_set_id')
-      expect(page).to have_select('generic_work_admin_set_id', selected: 'Default Admin Set')
+      pending("Admin sets no longer appear on the admin dashboard in the analytics rewrite. Not sure where admin set page will move to")
```

Comment:
Why does this test change in this Pull request. I'm not seeing the logic for this change here.  Am I missing it?

Reply:
At least in our analytics mockups. Admin Sets don't appear on the dashboard page for admins anymore. The test seemed like it was testing for the presence of something that's not going to be there.

Reply:
Did this behavior change in this PR? We shouldn't make test changes in a PR that isn't also changing the code that provides this behavior, because it's confusing if there isn't some correlation between where the tests change and where the behavior changes.